includes/db/Database.php

```diff
@@ -260,7 +260,7 @@ public function getServerInfo() {
 	 *   - false to disable debugging
```

Comment:
All the changes in this file are to update the doc comments, with the exception of one unused global variable I removed below.